src/BloomExe/web/EnhancedImageServer.cs

```diff
@@ -487,6 +487,12 @@ private bool ProcessAnyFileContent(IRequestInfo info, string localPath)
 					return true;
 				}
 			}
+			if (!File.Exists(localPath) && info.RawUrl.Contains("%25"))
```

Comment:
Could you add a comment explaining how we get to %25? I have two questions:
1) Is there an existing bug that leads to this (if so lets fix that)? Is this an old bug that left files in the wild with this problem? Let's say that.
2) Whatever the source, how are the urls then saved to disk? Are they saved as "My%20Dog.png" and then that becomes %2520... something like that? Can we add that to the comment?